
  0%|                                                                                                                                                                                           | 0/11 [00:05<?, ?it/s]
Traceback (most recent call last):
  File "/home/sehwan/MIIL/CA-layout-generation/dlt/inference.py", line 275, in <module>
    app.run(main)
  File "/home/sehwan/anaconda3/envs/CAL/lib/python3.9/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/home/sehwan/anaconda3/envs/CAL/lib/python3.9/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/home/sehwan/MIIL/CA-layout-generation/dlt/inference.py", line 189, in main
    collage = create_collage(true_geometry, id, geometry, canvas_size, base_path, config.scaling_size, config.mean_0)
  File "/home/sehwan/MIIL/CA-layout-generation/dlt/visualize.py", line 14, in create_collage
    background = Image.open(background_path)
  File "/home/sehwan/anaconda3/envs/CAL/lib/python3.9/site-packages/PIL/Image.py", line 3247, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'visualize_picture/Aesthetic Fashion Collection Presentation/Slide1_backgorund.png'